### ISSUE TITLE
CTRNN Issue in demo?

### ISSUE THREAD

**Issue by maxss280** [CodeReclaimers/neat-python#80]
I am trying to run the ctrnn demo in python 3.5 but it cannot seem to find the attribute ctrnn? 
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-9-0c6eadc7949e> in <module>()
     10 node2_inputs = [(1, -0.2), (2, 0.9)]
     11 
---> 12 node_evals = {1: neat.ctrnn.CTRNNNodeEval(0.01, sigmoid_activation, sum, -2.75 / 5.0, 1.0, node1_inputs),
     13               2: neat.ctrnn.CTRNNNodeEval(0.01, sigmoid_activation, sum, -1.75 / 5.0, 1.0, node2_inputs)}
     14 

AttributeError: module 'neat' has no attribute 'ctrnn'
```
I'm not sure what I am missing here because it appears to be there?

**Comment by d0pa** [comment]
What version of neat are you using ?

**Comment by maxss280** [comment]
neat-python-0.91 it looks like

**Comment by d0pa** [comment]
Does any of xor examples work ? I am guessing issue might be with "import neat".

**Comment by maxss280** [comment]
Yeah I was able to get some of the xor models to work. I had to update some of the config though it didn't like using __file__ for some reason for the path? That was all in Windows 10. I switched to my ubuntu environment and I can get everything to run fine same Python version and Neat-Python Version. 

Sounds like maybe local to my installation somehow. If no one else has issues on Windows. I'll just stay in Linux lol and play around with it XD thanks for your help!

**Comment by d0pa** [comment]
cool, looks like importing issue on windows, try adding import path manually in demo examples.
